src/foraging_gui/ForagingGUI.ui

```diff
@@ -79,7 +79,7 @@
            <x>0</x>
            <y>0</y>
            <width>894</width>
-           <height>236</height>
```

Comment:
Can you include screenshots of changes to the `.ui` files

Reply:
Updated in the description